features/_your_details/editing.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Your Details editing
+  In order to change my details,
+  As a user of the WPCC tool,
+  I want to be able to return to edit my choices.
+
+Background:
+  Given I am on the Your Details step
+
+Scenario Outline: on Your Contributions page
+  Given I am a "<age>" year old "<gender>"
+  And my salary is "<salary>" "<salary_frequency>" with "<contribution>" contribution
+  And I proceed to the next step
+  When I click to edit my details
+  Then I should return to the Your Details step
+  And I should see my "<age>", "<gender>", "<salary>", "<selected_frequency>" and "<contribution>" in the form
+
+  Examples:
```

Comment:
I'm not sure we need Welsh tests for this fix - its about the logic of how the form works and not about any specific text appearing.